llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -841,7 +843,7 @@ cc_library(
             "lib/Analysis/*.def",
         ],
         exclude = [
-            # TODO: Add these files to the build.
+            # TODO(mtrofin): Add these files to the build.
```

Comment:
Leaving these out was intentional -- I don't think this adds value in the open source version.

If we want to add LLVM bug links, that would work, but otherwise I'd just remove internal usernames.

Reply:
It's a github username too. We have TODO(gcmn) and TODO(chandlerc) floating around here, so this seemed ok

Reply:
Ok.... Not sure how we're going to keep that in sync, but sure.

Reply:
Well we will soon be deriving the internal version from upstream rather than the other way around. I'm writing the scripting right now and this seemed like a fine thing to have upstream and not something worth adding special handling to add on import :grin: